src/utils/components/ModalWithBackdrop.vue

```diff
@@ -226,6 +221,10 @@ export default {
             // But for desktop we let the size be dynamic with max to 90% of the view
             max-width: 80vw;
             max-height: 90svh;
+
+            &.modal-popup-compact {
+                width: 20vw;
```

Comment:
could you use the variable `$overlay-width` here? (from `variables.module.scss`)
This way it will be exactly the same size as the feature tooltip.

With relative width, it tends to break apart on mobile if you don't deal with smaller screen width separately

Reply:
ah ok, thanks!